용재/1406.py

```diff
@@ -0,0 +1,27 @@
+# https://www.acmicpc.net/problem/1406
+# 에디터
+
+getText = list(input())
+
+N = int(input())
+
+inputNumbers = [str(input()) for _ in range(N)]
```

Comment:
```suggestion
inputNumbers = [input() for _ in range(N)]
```
`input()`함수는 문자열을 반환하기 때문에 `str()`함수는 생략 가능할 거 같아요!